src/main.js

```diff
@@ -1,8 +1,12 @@
 import Vue from 'vue'
 import App from './App.vue'
+import router from './router'
+import resetcss from '@/css/resetCss.css'
```

Comment:
```suggestion
import '@/css/resetCss.css'
```
`reset-css.css`, or just simply `reset.css`, is a better name. Don't use camelCase for file names.